RecoCTPPS/PixelLocal/src/RPixDetClusterizer.cc

```diff
@@ -65,29 +65,31 @@ void RPixDetClusterizer::buildClusters(unsigned int detId, const std::vector<CTP
 //calibrate digi and store the new ones
       electrons = calibrate(detId,adc,row,column,pcalibrations);
       RPixCalibDigi calibDigi(row,column,adc,electrons);
-      calib_rpix_digi_set_.insert(calibDigi);
+      unsigned int index = column*maxRow + row;
+      calib_rpix_digi_map_.insert(std::pair<unsigned int, RPixCalibDigi> (index, calibDigi));
       if(calibDigi.electrons() > SeedADCThreshold_*ElectronADCGain_)
 	SeedVector_.push_back(calibDigi);
     }
   }
-  if(verbosity_) edm::LogInfo("RPixDetClusterizer")<<" RPix set size = "<<calib_rpix_digi_set_.size();
-  
+  if(verbosity_) edm::LogInfo("RPixDetClusterizer")<<" RPix set size = "<<calib_rpix_digi_map_.size();
+
   for(auto SeedIt : SeedVector_){
     make_cluster( SeedIt, clusters);
   }
 }
 
 void RPixDetClusterizer::make_cluster(RPixCalibDigi const &aSeed,  std::vector<CTPPSPixelCluster> &clusters ){
 /// check if seed already used
-  if(calib_rpix_digi_set_.size()==0 || calib_rpix_digi_set_.find(aSeed)==calib_rpix_digi_set_.end() ){
+  unsigned int seedIndex = aSeed.column()*maxRow + aSeed.row();
+  if(calib_rpix_digi_map_.empty() || calib_rpix_digi_map_.find(seedIndex)==calib_rpix_digi_map_.end() ){
```

Comment:
hi @fabferro - the check on .empty() is no longer needed here. (maybe it never was)

Reply:
Yes, "find" should do the job already